### PR TITLE
feature/AT-RESOURCES-22_GetAllStories

### PR DESCRIPTION
**Pull request by Bwore**

### CODE REVIEW

**Comment by ivanflores15** [comment]
You made incorrect changes on pom.xml.

`<groupId> **com.example** </groupId>
<artifactId> **demo** </artifactId>
<version> 0.0.1-SNAPSHOT </version>
<name> **demo** </name>
<description> **Demo project for Spring Boot** </description> `

Please don't touch those fields unless there is a good reason for it.
If you added new dependencies add them at end of section, in that way we could identify fastly which you added and avoiding conflicts with other teams.

File: pom.xml [hunk 1]
```diff
@@ -51,6 +51,7 @@
 			<groupId>org.projectlombok</groupId>
 			<artifactId>lombok</artifactId>
 			<version>1.18.12</version>
+			<artifactId>lombok</artifactId>
```
Comment (ivanflores15) [inline pom.xml]: This artifact is duplicated. It generates error bluid of project.

File: pom.xml [hunk 2]
```diff
@@ -86,6 +87,7 @@
 		</dependency>
 		<dependency>
 		   <groupId>io.springfox</groupId>
+			<groupId>io.springfox</groupId>
```
Comment (ivanflores15) [inline pom.xml]: This group is also replicated.

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
take a look on this please.

File: src/main/java/com/resources/controller/ResourcesController.java [hunk 3]
```diff
@@ -1,52 +1,52 @@
...
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.client.RestTemplate;
+import com.resources.model.Story;
+import org.springframework.beans.factory.annotation.Autowired;
```
Comment (ivanflores15) [inline src/main/java/com/resources/controller/ResourcesController.java]: Duplicated import

File: src/main/java/com/resources/controller/ResourcesController.java [hunk 4]
```diff
@@ -1,52 +1,52 @@
...
+import org.springframework.http.HttpStatus;
```
Comment (ivanflores15) [inline src/main/java/com/resources/controller/ResourcesController.java]: Duplicated import

File: src/main/java/com/resources/controller/ResourcesController.java [hunk 5]
```diff
@@ -1,52 +1,52 @@
...
+import com.resources.services.ResourcesService;
+
+import lombok.extern.slf4j.Slf4j;
+import io.swagger.annotations.ApiParam;
+
+@RequestMapping(value = "/at-resources-api/api/v1")
+@RestController
+@Slf4j
+public class ResourcesController {
+	RestTemplate restTemplate;
```
Comment (ivanflores15) [inline src/main/java/com/resources/controller/ResourcesController.java]: This variable is never use. Is it necessary?

File: pom.xml (hunk 2 above)
Comment (Bwore) [reply pom.xml]: I think I already solved it

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
Take a look on it, please.

File: src/main/java/com/resources/controller/ResourcesController.java (hunk 5 above)
Comment (Bwore) [reply src/main/java/com/resources/controller/ResourcesController.java]: Sorry, this variable was not necessary.

File: src/main/java/com/resources/dto/StoryDTO.java [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+import org.bson.types.ObjectId;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@Document(collection="stories")
```
Comment (ivanflores15) [inline src/main/java/com/resources/dto/StoryDTO.java]: This annotation is not necessary

File: src/main/java/com/resources/dto/StoryDTO.java [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+public class StoryDTO{
+
+	@Id
```
Comment (ivanflores15) [inline src/main/java/com/resources/dto/StoryDTO.java]: This annotation is not necessary

File: src/main/java/com/resources/dto/StoryDTO.java [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+	@GeneratedValue(strategy = GenerationType.AUTO)
```
Comment (ivanflores15) [inline src/main/java/com/resources/dto/StoryDTO.java]: This annotation is not necessary

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Review by rodrigo-velasco — APPROVED** [review]

**Review by ivanflores15 — APPROVED** [review]

**Pull request merged**